samples/bluetooth/central_gatt_write/src/gatt_write_common.c

```diff
@@ -256,6 +288,9 @@ static void connected(struct bt_conn *conn, uint8_t conn_err)
 		phy_update_countdown = PHY_UPDATE_COUNTDOWN;
 		phy_param_idx = 0U;
 	}
+
+	/* Offset param update by 1 metric collection after PHY update */
+	param_update_countdown = PARAM_UPDATE_COUNTDOWN + 1U;
```

Comment:
I don't understand the comment, can you elaborate? What is a metric collection?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/92572/files#diff-25e6d698734de351e15e26aa17ef532b57cbd3f591decaf457ab340eb16c44f0R141-R142

Every 1 second the accumulated data size is used for the throughput it calculation. PHY update is performed in reference to this calculation interval, and connection update is offset by 1 of this interval.